mod_ci/controllers.py

```diff
@@ -99,38 +100,28 @@ def start_platforms(db, repository, delay=None, platform=None) -> None:
 
     with app.app_context():
         from flask import current_app
+        app = current_app._get_current_object()
         if platform is None or platform == TestPlatform.linux:
-            linux_kvm_name = config.get('KVM_LINUX_NAME', '')
             log.info('Define process to run Linux VM')
```

Comment:
Let's update the log messages as well. For instance, in place of the message "... Linux VM", we may use "... Linux GCP instance".